parcels/scripts/check_corrupted_files.py

```diff
@@ -0,0 +1,35 @@
+"""
+Function to check if the data of a variable in a field list is corrupted
+"""
+
+import numpy as np
+import xarray as xr
+import os
+
+
+def check_corrupted_files(filelist, fieldname, outputfile=None):
+    """
+    parameter filelist: list of files which contain the field with name 'fieldname'
```

Comment:
call it `param` instead of `parameter`, to do as elsewhere in the code :)

Reply:
Done!